uni/lib/scan_util.icn

```diff
@@ -156,12 +156,13 @@ class FindFirst : Object(fchars,cMaps,subs,lastMatch)
    end
 
    #<p>
-   #  Move past the match, returning it.  Fails if no match yet.
+   #  Move past the match, returning it.  Fails if no match yet or if the
+   #  scan position has been moved since the last call to <tt>locate()</tt>.
    #   This is a matching procedure.
    #  <[returns matched substring]>
    #</p>
-   method moveMatch(s,i,j)
-      return 1(\lastMatch, move(*lastMatch,s,i,j))
+   method moveMatch()
```

Comment:
Is this a function introduced recently? It if is old and being used by other users we run the risk of breaking their code. Just making sure.

Reply:
If they're using this method, and the change breaks things, then they've been using it incorrectly.  The old implementation was faulty and could have led to unexpected behavior that was almost certainly not what was intended and would not have matched the behavior as described in the comments.